src/pages/ImportData.js

```diff
@@ -368,6 +377,16 @@ export const isProtected = (url, filetype) => {
   }
 };
 
+// This method identifies whether a workspace qualifies as protected.
+// 'Protected' here means that it has enhanced logging - either on its own or because it has an auth domain.
+// For now this also means only GCP workspaces are included.
+export const isProtectedWorkspace = (workspace) => {
+  if (!isGoogleWorkspace(workspace)) {
+    return false;
+  }
+  return !!workspace.workspace.bucketName && workspace.workspace.bucketName.startsWith('fc-secure');
```

Comment:
I agree - I would prefer an official flag.  It would most likely be determined the same way, but would have to be implemented in the rawls workspace model,